src/context/providers/PlanewaveCutoffsContextProvider.js

```diff
@@ -35,11 +40,67 @@ export class PlanewaveCutoffsContextProvider extends mix(ContextProvider).with(
         return cutoffConfig[this.application.name];
     }
 
+    /**
+     * @summary Find cutoff values corresponding to wavefunction or charge
+     * density with accuracy level in the pseudo method data
+     * @param data {Object}: the data object from pseudo method data
+     * @param cutoffEntity {String}: "wavefunction", or "density"
+     * @param accuracyLevel {String}: "standard", "high", or "low"
+     * @return {number}: if cutoff value present returns value else return 0
+     */
+    _cutoffFromPseudoData = (data, cutoffEntity, accuracyLevel = "standard") => {
```

Comment:
This could be called - getCutoffFromMethodData to make it generic. Then inside PseudopotentialMethod redefined to use pseudo per the below.